examples/seal/unseal.c

```diff
@@ -28,6 +28,7 @@
 #include <examples/seal/seal.h>
 #include <examples/tpm_io.h>
 #include <examples/tpm_test.h>
+#include <examples/tpm_test_keys.h>
```

Comment:
Getting a linker error. You'll need to add `tpm_test_keys.c` to the examples/seal/include.am:

```
wolfTPM % make
/Applications/Xcode.app/Contents/Developer/usr/bin/make -j17  all-am
  CCLD     examples/seal/unseal
Undefined symbols for architecture x86_64:
  "_getPrimaryStoragekey", referenced from:
      _TPM2_Unseal_Example in unseal.o
  "_readKeyBlob", referenced from:
      _TPM2_Unseal_Example in unseal.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Reply:
Fixed